crowns.lic

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
💡 Needs documentation header

Reply:
I will add tonight. This was “quick” for some to test. Formayor is one tester.

Reply:
Added. Already added to Lich Repo scripting reference too.